CHANGELOG.md

```diff
@@ -36,6 +36,7 @@
 
 ### Bug Fixes
 
+- loader: gracefully handle ELF files with unsupported architectures @kami922 #2793
```

Comment:
Please move this to the unreleased section and change the issue number to the number of this PR.